### ISSUE TITLE
Question regarding channel attention module.

### ISSUE THREAD

**Issue by seanko29** [arp95/cbam_cnn_architectures_image_classification#1]
Hello there!

I was looking at VGG16 with CBAM.

From the forward function, I see  the following lines of codes

```
def forward(self,x):
    x = self.features_1(x)
    scale = torch.nn.functional.sigmoid(self.max_pool_1(x) + self.avg_pool_1(x)).expand_as(x)
    x = x * scale
    scale = torch.cat((torch.max(x, 1)[0].unsqueeze(1), torch.mean(x, 1).unsqueeze(1)), dim=1)
    scale = self.spatial_attention(scale)
    x = x * scale
```
![image](https://user-images.githubusercontent.com/72428646/222088432-8ecccb7d-9800-4b41-8e4d-1d53f9fd99b7.png)

I do not see shared MLP part of the code. I see that you get the maxpool and average pool of the input feature but you added sigmoid right away on top of shared MLP.

Can you explain where the shared MLP part is in the lines of codes of the forward function? 

Thank you in advanced :).
